### ISSUE TITLE
Type for `TitleField` missing `direction` field (mismatch in typings versus data from your endpoint)

### ISSUE THREAD

**Issue by maapteh** [prismicio/prismic-client#331]
### Versions

- @prismicio/client: 7.3.1
- node: 20.8.0

### Reproduction

In the data from the [cdn/api](https://prismic.io/docs/api#the-six-prismic-api-endpoints) we get a `direction` field in the data for `headings` and `paragraphs`, while in the Type definition this is not typed at all:

<img width="508" alt="Screenshot 2023-12-27 at 10 54 55" src="https://github.com/prismicio/prismic-client/assets/25961356/df0cdf2a-a411-48c7-99d4-ba6550254bc2">

### Steps to reproduce

Consume `Query` from '@prismicio/client' then browse to the field `TitleField`

There you will see:

```
/**
 * A title field.
 *
 * @see Title field documentation: {@link https://prismic.io/docs/rich-text-title}
 */
export type TitleField<State extends FieldState = FieldState> = State extends "empty" ? [] : [
    Omit<RTHeading1Node | RTHeading2Node | RTHeading3Node | RTHeading4Node | RTHeading5Node | RTHeading6Node, "spans"> & {
        spans: [];
    }
];
```

_Now add a paragraph and heading to your workspace and consume the Query endpoint. Inspect the data, you will notice the direction._

### What is expected?

The direction to be typed as well. Inside your definition of a textNodeBase for example or the specific heading and paragraph (i don't know your internals, so i assume it belongs in the RRText definition :)):

```
export interface RTTextNodeBase {
    text: string;
    spans: RTInlineNode[];
    direction: 'ltr' | 'rtl' whatever :)
}
```

### What is actually happening?

Not typed, so my type builders are unable to use auto generated typings. Please add them correctly according to your own data schema. At least for Paragraph and Headings but just check your schema one more time :)

**Comment by lihbr** [comment]
Will be fixed in #342!

**Comment by lihbr** [comment]
Released as part of [v7.7.0](https://github.com/prismicio/prismic-client/blob/master/CHANGELOG.md#770-2024-07-23) 🎉
